src/KnxDevice.cpp

```diff
@@ -336,8 +338,7 @@ template e_KnxDeviceStatus KnxDevice::write <double>(byte objectIndex, double va
  */
 e_KnxDeviceStatus KnxDevice::write(byte objectIndex, byte valuePtr[]) {
     type_tx_action action;
-    type_tx_action action2;
-    byte *dptValue;
+    //type_tx_action action2;
```

Comment:
can be removed completely, not just disable by comment.

Reply:
Sure, but it is used in code which is also commented out a few lines later. I did not want to break this.